index.html

```diff
@@ -1,111 +1,579 @@
+              <div class="post-title">
+                <h3 class="post-title-name">Sonic live</h3>
+                <p class="post-content">
+                  One September 22, 2021 A volutpat nulla eros, nulla ultrices
+                  nec nulla imperdict
+                </p>
+              </div>
+              <a class="link" href="blog.html">Read more</a>
             </div>
-
-            <div class="body">
-              <div class="profile-icon-box">
-                <div class="profile-icon">
-                    <img src="https://assets.codepen.io/4292193/ayakoyokoya_icon2.png" />
-                </div>
+            <div class="mini-post">
+              <img src="#" alt="text" />
+              <span class="post-lable">NTF</span>
+              <div class="post-info">
+                <span class="post-author">Tony Nguyen</span>
+                <span class="post-date">20 Jun 2022</span>
               </div>
-
-              <div class="name">
-                <p>error!!!!</p>
-                <p>error!!!!</p>
-                <p>error!!!!</p>
-                <p>error!!!!</p>
-                <p>error!!!!</p>
-                <button class="ok-error-btn" onclick="window.location.reload();">OK...</button>
+              <div class="post-title">
+                <h3 class="post-title-name">Sonic live</h3>
+                <p class="post-content">
+                  One September 22, 2021 A volutpat nulla eros, nulla ultrices
+                  nec nulla imperdict
+                </p>
               </div>
+              <a class="link" href="blog.html">Read more</a>
+            </div>
+          </article>
+          <article class="blog-post">
+            <img src="#" alt="text" />
+            <span class="post-lable">NTF</span>
+            <div class="post-info">
+              <span class="post-author">Tony Nguyen</span>
+              <span class="post-date">20 Jun 2022</span>
             </div>
+            <div class="post-title">
+              <h3 class="post-title-name">Sonic live</h3>
+              <p class="post-content">
+                One September 22, 2021 A volutpat nulla eros, nulla ultrices nec
+                nulla imperdict
+              </p>
+              <a class="link" href="blog.html">Read more</a>
+            </div>
+          </article>
+        </div>
+        <button type="button">Read more</button>
+      </section>
+      <section class="about-as">
+        <h2 class="about-as-title">About us</h2>
+        <p class="about-as-text">
+          Duls aute irure dolor in reprehenderit in voluptate velit esse cillum
+          dolore eu fugiat nulla pariatur. Excepteur sint occae cat cupidatat
+          not prodient, sunt in culpa qui officia dese runt molit anim id est
+          laborum velit esse cillium dolore eu fugiat nulla pariatu epteur sint
+          occaecat.
+        </p>
+        <div class="about-as-description">
+          <h3 class="about-as-list">High quality</h3>
```

Comment:
You need to combine next 2 blocks into 1
<img width="230" alt="Screenshot 2024-11-11 at 7 32 09 PM" src="https://github.com/user-attachments/assets/22d085bb-4bd2-4189-b386-6e538bf125da">